### PR TITLE
Created solution for weird-string-case

### PR DESCRIPTION
**Pull request by mpsujek**
I think I need to read this cata instruction tomorrow to understand what has to be done there.

### CODE REVIEW

File: js/weird-string-case.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// https://www.codewars.com/kata/52b757663a95b11b3d00062d/train/javascript
+
+function toWeirdCase(string){
+  const newArr = string.split('').map((letter, i) => {
+      if (letter === ' '){
+          i+1;
```
Comment (Gelio) [inline js/weird-string-case.js]: What does this do?

File: js/weird-string-case.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const toWeirdCase = require('./weird-string-case');
+
+describe('weird string case', () => {
+
+  test('Got string with spaces should return in weird case', () => {
```
Comment (Gelio) [inline js/weird-string-case.test.js]: The test's full name will be:

> weird string case Got string with spaces should return in weird case

Does not look like a proper sentence 😄 Please adjust it

File: js/weird-string-case.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// https://www.codewars.com/kata/52b757663a95b11b3d00062d/train/javascript
+
+function toWeirdCase(string){
+  const newArr = string.split('').map((letter, i) => {
```
Comment (Gelio) [inline js/weird-string-case.js]: How about returning the result immediately? It's not needed to have this variable, you could just return the chain of operations

File: js/weird-string-case.js (hunk 1 above)
Comment (psujek) [reply js/weird-string-case.js]: Now I see it do nothing.

**Pull request merged**